pkg/controller/chaosengine/chaosengine_controller.go

```diff
@@ -218,29 +204,67 @@ func (r *ReconcileChaosEngine) Reconcile(request reconcile.Request) (reconcile.R
 		reconcileEngine: engineReconcile,
 	}
 
+	// Check if the engineRunner pod already exists, else create
+	err = engineRunnerPod(runnerPod)
+	if err != nil {
+		return reconcile.Result{}, err
+	}
+	if engine.instance.Spec.Monitoring {
+		reconcileResult, err := createMonitoringResources(engine, engineReconcile)
+		if err != nil {
+			return reconcileResult, err
+		}
+	} else {
+		reqLogger.Info("Monitoring is DISABLED")
+	}
+
+	return reconcile.Result{}, nil
+}
+
+//
+func createMonitoringResources(engine engineInfo, recEngine *reconcileEngine) (reconcile.Result, error) {
+	//reqLogger.Info("Monitoring is Enabled")
+	// Define the engine-monitor service which is secondary-resource #2
+	engineMonitor, err := newExporterServiceForCR(engine)
```

Comment:
Change the variable: `engineMonitor` --> `engineExporter`

Reply:
Do one thing, It's a little bit confusion around exporter and monitor, Change every word `exporter` to `monitor`, Stick to one word